fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -222,6 +226,7 @@ protected FDBRecordStore(@Nonnull FDBRecordContext context,
 
         this.omitUnsplitRecordSuffix = formatVersion < SAVE_UNSPLIT_WITH_SUFFIX_FORMAT_VERSION;
         this.preloadCache = CacheBuilder.<Tuple,FDBRawRecord>newBuilder().maximumSize(PRELOAD_CACHE_SIZE).build();
+        this.localMetaDataFileDescriptor = localMetaDataFileDescriptor;
```

Comment:
This should go next to the initialization of `metaDataProvider` in order to match the order in which the parameters are given in the constructor.

Reply:
sure.